rfcs/20201023-sig-models.md

```diff
@@ -0,0 +1,91 @@
+# Creating SIG Models
+
+| Status        | Proposed                                             |
+:-------------- |:---------------------------------------------------- |
+| **RFC #**     |                                                      |
+| **Author(s)** | Jaeyoun Kim (jaeyounkim@google.com), Jing Li (jingli@google.com), Mike Liang (mliang@google.com), Shuangfeng Li (shuangfeng@google.com) |
+| **Sponsor**   | Thea Lamkin (thealamkin@google.com)                  |
+| **Updated**   | 2020-10-23                                           |
+
+## What is this group for?
+
+This group is for discussions and collaborations on enabling community
+contributions to [TensorFlow Model Garden](https://github.com/tensorflow/models)
+and [Tensorflow Hub](https://github.com/tensorflow/hub).
+
+SIG Models will focus on empowering the community to contribute state-of-the-art
+model implementation in TensorFlow 2. It will benefit the whole community by
+providing recommended implementations and models with reproducible results.
+
+SIG Models will have several subgroups (e.g., SIG Models Vision and SIG Models
+NLP) covering different machine learning areas. There are several SIG leads for
+each group to coordinate the contributions, run community events like contests,
+and maintain the code quality through the review process. Each group has the
+flexibility to operate differently.
+
+SIG is also a place for community discussions and sharing best practices of
+using TensorFlow 2 for state-of-the-art research. Furthermore, SIG provides
+product feedback to help TensorFlow to be improved.
+
+## Who will be part of it?
```

Comment:
What about `keras-cv/keras-nlp`?

Reply:
Same question.

Reply:
@bhack @AakashKumarNain Yes, the Keras team (keras-cv/nlp) will contribute to the SIG Models.